nemo_rl/environments/simulated_user/unique_numbers.py

```
@@ -0,0 +1,296 @@
+                grading_response = asyncio.run(
+                    run_prompt_async(
+                        metadata["grader_runner"],
+                        "grader",
+                        grading_prompt,
+                        silence=True,
+                    )
+                )
+                try:
+                    grade = int(re.search(r"(\d+)", grading_response).group(1))
+                    reward = (reward + grade) / 2.0
+                except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)